### ISSUE TITLE
Identify individual for reply within a group

### ISSUE THREAD

**Issue by jhoc2000** [kontalk/androidclient#1108]
### Expected behavior
If one is chatting within a group one may wish to address a response to an individual within that group but also want the rest of the group to see the response. 
### Actual behaviour 
One can't identify an individual to reply to within a group. For example, a discussion within a family group. I want to ask one of those people within that group what they want for their birthday. It's only relevant to that person in the group but the others within that group would still like to see the response. 
### Steps to reproduce
1. Create a Kontalk group. 
2. Start discussion with that group. 
3. One can't direct a response to an individual within that group and also let the others see that response. 
4. If one could long press on a person's identifier within that group then one could direct an answer to that person within that group and it would be identified by an @individual for the others in that group to see. 
For example, 

C to group: shall we get together for A's birthday?
A's response: I'm free Saturday.
B's response: I can make it for Saturday's lunch but not dinner. 
C's response: okay. Let's meet at my place for lunch on Saturday. 
B's @ A: what do you want for your birthday? 
### Environment

Kontalk version: (please also specify origin, e.g. F-Droid or Google Play)
Kontalk 4.1.0-beta4 from F-Droid 
Android version: (please also specify custom ROM name if applicable)
Android 6.0.1
Device model:
Sony Xperia Z2 

### Logs

If you have a system log related to the issue please send it via email at
devteam (at) kontalk.org. Please do not attach it or publish it anywhere because
it may contain sensitive information. If you have root access, you can use Catlog
(available on both Google Play and F-Droid) to produce it. Alternatively, if you
have the Android SDK installed on your computer, connect your device and execute:

```
adb logcat -d | grep `adb shell ps | grep org.kontalk | cut -c10-15` >kontalk-log.txt
```

and send us the text file.

### Other

If there is a relevant forum discussion, please link to it here.

**Comment by daniele-athome** [comment]
Thanks for suggesting! I believe this can be achieved through use of mentions:

> Alice: @Bob what do you want for your birthday?

This will make possible to mention multiple users and also integrate mentions in the sentence gracefully.

**Comment by jhoc2000** [comment]
Yes. That looks good. It also allows the others in the group to see what bob wants.

**Comment by OffifialMITX** [comment]
The Riot.im app hadles this by using a ":". For me "Bob: [...]" feels more natural than "@Bob [...]" because in a rl conversation you normally speak to the person who you like to recieve your message's content rather than staring blankly at some random point and just mentioning who the reciever is. But hey, thatay be just me. (I'd like to mention though that nobody I know who's using Kontalk is used to GitHub)

On November 28, 2017 10:52:58 AM GMT+01:00, jhoc2000 <notifications@github.com> wrote:
>Yes. That looks good. It also allows the others in the group to see
>what bob wants. 
>
>On 28 November 2017 8:40:53 PM AEDT, Daniele Ricci
><notifications@github.com> wrote:
>>Thanks for suggesting! I believe this can be achieved through use of
>>mentions:
>>
>>> Alice: @Bob what do you want for your birthday?
>>
>>This will make possible to mention multiple users and also integrate
>>mentions in the sentence gracefully.
>>
>>-- 
>>You are receiving this because you authored the thread.
>>Reply to this email directly or view it on GitHub:
>>https://github.com/kontalk/androidclient/issues/1108#issuecomment-347467136
>
>
>-- 
>You are receiving this because you are subscribed to this thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/kontalk/androidclient/issues/1108#issuecomment-347470475

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by jhoc2000** [comment]
I got the idea from WeChat. One just long presses on the image or identifier of the person in the group to whom one wants to direct his or her comment and the WeChat app automatically inserts the "@bob" text. One then types in his or her comment for Bob. 

I'm used to the @..... identifier and, for me, it also fits into a group situation. That is, "this comment is directed at bob" rather than "this comment is directed to bob". The latter may make the others in the group feel that they are listening in on a private message. Whereas, the former indicates that all can listen but the comment is at bob. 

That's just my two cents worth and of course I'm open to the majority's decision.
